initial changes to make twonav work (installing their packages)

eboschtwonav opened the pull request.

eboschtwonav committed:
initial changes to make twonav work (installing their packages)

iserram commented on make_var_mx6ul_dart_debian.sh:
```diff
@@ -87,7 +85,7 @@ readonly G_EXT_CROSS_COMPILER_NAME='gcc-linaro-4.9-2016.02-x86_64_arm-linux-gnue
 ############## user rootfs packages ##########
-readonly G_USER_PACKAGES=""
+readonly G_USER_PACKAGES="libc6 libglu1-mesa-dev libcurl4-gnutls-dev libsdl1.2-dev libsdl-mixer1.2-dev libsdl-ttf2.0-dev libfreetype6-dev libiw-dev libapt-pkg-dev libbluetooth-dev dosfstools gdbserver"
 
```
Totes les dependencies de TN i llibreries estan resoltes?

eboschtwonav replied:
no, estan resoltes les dependencies prèvies. Cal instal·lar (i recompilar) tots els paquets de TwoNav. Aquí es contemplen les dependències necessàries per una imatge "base" (Sense TwoNav)

eboschtwonav committed:
missing links for libWifi

eranmati committed:
linux: Update to latest revision

Include the following commits:

a0b5a61905e dart-6ul: Remove obsolete comment from DT
ca6dc3e39ed dart-6ul: Reducing the speed of SDHC1 clock to 132Mhz
9fe9500c07b var-som-mx7: ov5640_mipi: Set correct clk and remove unattached gpios
941db7f7fad VAR-SOM-MX7: Disabled SDIO Bluetooth support

Signed-off-by: Eran Matityahu <eran.m@variscite.com>

Commit:
linux: Update to latest revision

Include the following commits:

76bd3bfa41d var-som-mx7: Reworked WIFI configuration in DTS
37b06f360f6 imx6ul-var-dart: Reworked WIFI configuration in DTS
2c83fe7c93a rtc: dart-6ul: Added RTC alarm IRQ support for DART-6UL external RTC
806e0515dff rtc: Variscite: Fixed external RTC module identification
4bf42694398 rtc-ds1307: Make GPIO connected alarm IRQ wake-up the system from suspend to RAM.
8640787b412 var-som-mx7: ov5640_mipi: migrate camera fix from rocko

Commit:
u-boot: Update to latest revision

Include the following commits:

724493a7c85 mx6var_som: backup silence status before calling mmc_init
535519be584 MLK-14945 HAB: Check if IVT valid before authenticating image
888f7c2766c MLK-16703: HAB : Check if CSF is valid before authenticating image
1dad2408add mx6var_som: Move the setup_display() call from board_early_init_f() to board_init()
4c9eadeffc0 mx6ul_var_dart: Update get_cpu_speed_grade_hz for i.MX6ULL
a3ff0978990 mx6ul_var_dart: Correct get_cpu_speed_grade_hz on 696MHz SoCs
2f08c15a484 compiler*.h: sync include/linux/compiler*.h with Linux 4.5-rc6
a71497c73fe Revert "Fix the compile issue under gcc6"

Commit:
Bluetooth: Updated variscite-bluetooth script

Commit:
WIFI: Added bcm43xx WIFI initialization startup script

eboschtwonav committed:
unwind libs included (to get backtrace)

eboschtwonav committed:
Merge remote-tracking branch 'origin/debian_jessie_mx6ul_var03' into rootfs_modifications

# Conflicts:
#	make_var_mx6ul_dart_debian.sh

pguillamontwonav commented on variscite/variscite-wifi:
```diff
@@ -0,0 +1,184 @@
+
+		# BT_EN down
+		echo 0 > /sys/class/gpio/gpio${BT_EN_GPIO}/value
+	fi
```
I think this if-else statement can be simplified as follows:
```
if som_is_dart_6ul_5g; then
	# WIFI power up
	echo 1 > /sys/class/gpio/gpio${WIFI_PWR_GPIO}/value
	#usleep 10000
	sleep 0.01
fi

# WLAN_EN up
echo 1 > /sys/class/gpio/gpio${WIFI_EN_GPIO}/value

# BT_EN up
echo 1 > /sys/class/gpio/gpio${BT_EN_GPIO}/value

# Wait 150ms at least
#usleep 200000
sleep 0.2

# BT_EN down
echo 0 > /sys/class/gpio/gpio${BT_EN_GPIO}/value
```
Since the only difference is enabling `gpio${WIFI_PWR_GPIO}/value` in the `som_is_dart_6ul_5g` block.

Do you think we can do it like this? Is there any other thing I am not seeing?

pguillamontwonav replied:
I would apply the same in the function `wifi_down()`

lluisdiaz commented on make_var_mx6ul_dart_debian.sh:
```diff
@@ -87,7 +85,7 @@ readonly G_EXT_CROSS_COMPILER_NAME='gcc-linaro-4.9-2016.02-x86_64_arm-linux-gnue
 
 ############## user rootfs packages ##########
-readonly G_USER_PACKAGES=""
+readonly G_USER_PACKAGES="libc6 libglu1-mesa-dev libcurl4-gnutls-dev libsdl1.2-dev libsdl-mixer1.2-dev libsdl-ttf2.0-dev libfreetype6-dev libiw-dev libapt-pkg-dev libbluetooth-dev dosfstools gdbserver libelf1 libdw1 libelf-dev libdw-dev"
```
afegeix uuid-dev -> compilar llibreria ant+

twonav replied:
no entenc lo de compilar la llibreria de ant+. Que es necessita el paquet per compilar o que afejeixi la compilacio de ant en el proces?

lluisdiaz replied:
sí, es necessita uuid-dev per compilar ant

eboschtwonav replied:
fet

lluisdiaz commented on make_var_mx6ul_dart_debian.sh:
```diff
@@ -499,15 +490,23 @@ rm -f user-stage
-
-	install -m 0644 ${G_VARISCITE_PATH}/wallpaper_800x480.png \
-		${ROOTFS_BASE}/usr/share/images/desktop-base/default
+	install -m 0644 ${G_VARISCITE_PATH}/uuid.h ${ROOTFS_BASE}/usr/include/bluetooth/
```
Això no m'agrada. aquest .h no forma part de la versió que porta  variscite de bluez. Només està aquí perquè si no twonav no va.

twonav replied:
aquest arxiu es el resultant de uuid-dev? si afegim el paquet podem eliminar el .h?

lluisdiaz replied:
no.. uuid-dev instala includes a /usr/include/uuid/

tpaschidis reviewed (dismissed):
I believe you :)

eboschtwonav committed:
uuid-dev added

eboschtwonav committed:
Update variscite-wifi
